docs/tutorial_bracket_strategy.md

```diff
@@ -97,23 +101,23 @@ Hence, it would be unwise to transfer more ETH than required for deployment into
-- _currentPrice_: Please provide the current price of the trading pair: [ targetToken ] / [ stableToken ]. Your input will be checked for plausibility by the script via the price of dex.ag
+- _baseTokenId_: You are specifying two tokens you want to provide liquidity for, the baseToken and the _quoteTokenId_ via their indices. In order to get the index of a token, follow the next section. If you specify the more stable token via quoteTokenId and the more volatile token via baseToken, then most likely the price parameter is more intuitive. Eg., if you are trading `ETH/DAI` with `stableToken=Index(DAI)`, then the parameter `currentPrice` can be specified as a number > 1.
+- _quoteTokenId_: specifies the second token you want to trade via its index. In order to determine the index, see the [section](####-Getting-the-index-of-your-tokens).
+- _depositBaseToken_: This is the cumulative sum of all baseTokens you want to invest in all brackets
+- _depositQuoteToken_: This is the cumulative sum of all quoteTokens you want to invest in all brackets
+- _currentPrice_: Please provide the current price of the trading pair: [ baseToken ] / [ quoteTokenId ]. Your input will be checked for plausibility by the script via the price of dex.ag
 - _lowestLimit_: The liquidity provided will be split over the a price range of [lowestLimit, highestLimit]. Hence, the lowestLimit specifies the lowest price any bracket should trade.
 - _highestLimit_: The liquidity provided will be split over the a price range of [lowestLimit, highestLimit]. Hence, the highestLimit specifies the highest price any bracket should trade.
 
-#### Getting the `tokenIndex` of your tokens
+#### Getting the index of your tokens
 
-All token that have been added to the Gnosis protocol get a unique `tokenIndex`. In order to read the `tokenIndex`, visit [Etherscan GP contract(Rinkeby)](https://rinkeby.etherscan.io/address/0xc576ea7bd102f7e476368a5e98fa455d1ea34de2#code) or for mainnet [Etherscan GP contract(Mainnet)](https://etherscan.io/address/0x6f400810b62df8e13fded51be75ff5393eaa841f). Then, click on the tab: _Contract_ and the button "Read Contract":
+All token that have been added to the Gnosis protocol get a unique index. In order to read the index, visit [Etherscan GP contract(Rinkeby)](https://rinkeby.etherscan.io/address/0xc576ea7bd102f7e476368a5e98fa455d1ea34de2#code) or for mainnet [Etherscan GP contract(Mainnet)](https://etherscan.io/address/0x6f400810b62df8e13fded51be75ff5393eaa841f). Then, click on the tab: _Contract_ and the button "Read Contract":
 
 ![Read Etherscan contract](assets/tutorial_bracket_strategy-etherscan-read.png)
 
-Then go down to the function: `tokenAddressToIdMap` and paste the token address of your token into the parameter field and run the query.
+Then go down to the 27.th function: `tokenAddressToIdMap` and paste the token address of your token into the parameter field and run the query.
```

Comment:
```suggestion
Then go down to the 27th function: `tokenAddressToIdMap` and paste the token address of your token into the parameter field and run the query.
```

Reply:
Maybe check in point number 1. from the add token tutorial [here](https://docs.gnosis.io/protocol/docs/addtoken5/) how the reference the item number.